### ISSUE TITLE
Unable to access download widget on laptop with 1366x768 resolution screen

### ISSUE THREAD

**Issue by plater** [adlerluiz/ytmdesktop#1453]
### Consent

- [x] I verified that there is no open/closed issue for the same subject.
- [x] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

cat /usr/lib/os-release
NAME="openSUSE Leap"
VERSION="15.6"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.6"
PRETTY_NAME="openSUSE Leap 15.6"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.6"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
DOCUMENTATION_URL="https://en.opensuse.org/Portal:Leap"
LOGO="distributor-logo-Leap"

I'm a multimedia packager and maintainer and I've created a package (doesn't build in the build service due to restriction on downloading dependencies) from git and have tried release v2.0.5 and now git head as well as the prebuilt rpm but the issue  persists.
My desktop work station with a 1600x900 resolution works

In a normal maximized or restored  window  I'm unable to access the widgets at the bottom of the screen including the download widget

![Image](https://github.com/user-attachments/assets/cbc4c40c-326f-41fc-a46c-971e8c7b7839)

but I am able to access them in full screen mode.

![Image](https://github.com/user-attachments/assets/0f5366b0-36e5-4571-86a4-725d673bd70e)

### Expected Behavior

My desktop pc works as expected with a 1600x900 resolution. Unfortunately my pc doesn't offer a 1366x768 resolution to test with.

![Image](https://github.com/user-attachments/assets/989318e0-bf77-4368-9213-7333083e68a8)

### Steps To Reproduce

Go to library and click on a playlist

### YTMDesktop

Developement or v2.0.5

### OS

Linux

### OS Version

Leap:15.6

### Arch

x64

### Installation way

System built rpm

**Comment by Alipoodle** [comment]
You are displaying 2 different applications in the pictures 🤔

Pictures 1&2 aren't this app.

**Comment by plater** [comment]
This is embarrassing, when I moved from spotify to youtube for my music I first tried a few players and created a menu entry for a ytmdesktop prebuilt expanded in my home directory on the laptop. I thought I had deleted the directory but although I have a .desktop file in /usr/share/applications it seems that my manual entry stuck.
In mitigation there are too many desktop youtube players.
If I run the app from a virtual console as "/usr/lib/youtube-music-desktop-app/youtube-music-desktop-app" I have the correct app which incidentally runs and starts much better than the foreign app
